### ISSUE TITLE
About CNNIQA

### ISSUE THREAD

**Issue by hazelAutumn** [chaofengc/IQA-PyTorch#91]
Hi,
Did you train CNN IQA on the full image 512x384 instead of splitting them into patches?
I could not find the code of splitting the model.

Also, I think there are issues with the pre-trained model of CNNIQA as I tested it on the test set of KonIQA and all the score values are in the range from [-5; 2] instead of [0, 100] although the training data and the test data are from the same database.

**Comment by chaofengc** [comment]
You may find the training information in the [option file](https://github.com/chaofengc/IQA-PyTorch/blob/main/options/train/CNNIQA/train_CNNIQA.yml)

The model is trained with $384\times384$ patches:

https://github.com/chaofengc/IQA-PyTorch/blob/e23bce2f60db9ce38a760236164f1e794a824fa3/options/train/CNNIQA/train_CNNIQA.yml#L17-L20

The score range is messed up because I used the PLCC loss for faster convergence:

https://github.com/chaofengc/IQA-PyTorch/blob/e23bce2f60db9ce38a760236164f1e794a824fa3/options/train/CNNIQA/train_CNNIQA.yml#L64-L66

**Comment by hazelAutumn** [comment]
Thanks for you quick response!
